style.css

```diff
@@ -29,7 +29,75 @@ article {
 	margin-bottom: 10px;
 	background-color: aqua;
 	width: 50%;
-	height: 150px;
+	height: 600px;
+}
+
+/* Article 1 */
+.article1 {
+	background-image: url(./img/alex/Moscow-Android-Wallpapers.jpg);
+	background-repeat: no-repeat;
+	background-size: 100%;
```

Comment:
if you change here as  background-size: 100% 100%; then it will cover your box for all devices perfctly

Reply:
yes, this works perfectly!thanks!